tests/intmap-properties.hs

```diff
@@ -966,6 +980,18 @@ prop_lookupLE = test_lookupSomething lookupLE (<=)
 prop_lookupGE :: [(Int, Int)] -> Bool
 prop_lookupGE = test_lookupSomething lookupGE (>=)
 
+prop_lookupMin :: [(Int, Int)] -> Property
+prop_lookupMin ys = length ys > 0 ==>
+  let xs = List.nubBy ((==) `on` fst) ys
+      m  = fromList xs
+  in  lookupMin m == Just (List.minimumBy (comparing fst) xs)
```

Comment:
I don't like these properties. They are confusing, and also inefficient. How about this?

```haskell
prop_lookupMin :: IntMap Int -> Property
prop_lookupMin im = lookupMin m === listToMaybe (toAscList m)
```

Reply:
Yes, that is much better.